test/e2e/servicebindingrequest_test.go

```diff
@@ -183,3 +191,45 @@ func serviceBindingRequestTest(t *testing.T, ctx *framework.TestCtx, f *framewor
 	_ = f.Client.Delete(todoCtx, &sbrSecret)
 	_ = f.Client.Delete(todoCtx, &d)
 }
+
+func CreateDB(todoCtx context.Context, t *testing.T, ctx *framework.TestCtx, f *framework.Framework, ns string, resourceRef string, secretName string) pgv1alpha1.Database {
```

Comment:
Would be nice to leave documentation comments on those methods as well, so we can assess what they are doing and use that description on composing test cases.

Reply:
Good point @otaviof, thanks for that - fixed.